runtime/jcl/common/stdinit.c

```diff
@@ -195,11 +195,11 @@ standardInit( J9JavaVM *vm, char *dllName)
 				if (NULL == clz) {
 					goto _fail;
 				}
-#if JAVA_SPEC_VERSION >= 17
+#if (17 <= JAVA_SPEC_VERSION) && (JAVA_SPEC_VERSION <= 18)
```

Comment:
I think this can be simplified w/ `#if JAVA_SPEC_VERSION < 19` since Java 15/16 are out of service.